packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.vue

```diff
@@ -0,0 +1,261 @@
+
+const { densityClass } = useDensity(props);
+
+/**
+ * Current value (with getter and setter) that can be used as "v-model" for the native input.
+ */
+const value = computed({
+  get: () => props.modelValue,
+  set: (value) => emit("update:modelValue", value),
+});
+
+const handleChange = (event: Event) => {
+  const inputValue = (event.target as HTMLInputElement).valueAsNumber;
+  emit("update:modelValue", inputValue);
+};
+
+const handleCounterButton = (event: Event, operation: "add" | "substract") => {
+  event.preventDefault();
+  if (props.disabled || props.readonly || props.loading) return;
+
+  if (operation === "add") {
+    const newValue = value.value + props.step;
+
+    if (props.max && newValue > props.max) return;
+    value.value = value.value + props.step;
+  } else {
+    const newValue = value.value - props.step;
+
+    if (props.min && newValue < props.min) return;
+    value.value = value.value - props.step;
+  }
+};
+</script>
+
+<template>
+  <div v-if="props.skeleton" :class="['onyx-stepper-skeleton', densityClass]">
+    <OnyxSkeleton v-if="!props.hideLabel" class="onyx-stepper-skeleton__label" />
+    <OnyxSkeleton class="onyx-stepper-skeleton__input" />
+  </div>
+  <div v-else :class="['onyx-stepper', densityClass]">
+    <OnyxFormElement v-bind="props" :error-messages="errorMessages">
+      <div class="onyx-stepper__wrapper">
+        <span
+          tabindex="0"
+          role="button"
+          class="onyx-stepper__counter"
+          :class="{ 'onyx-stepper__counter--disabled': props.min && props.min === value }"
+          :aria-label="t('substract')"
+          @click="handleCounterButton($event, 'substract')"
+          @keydown.enter="handleCounterButton($event, 'substract')"
+        >
+          <OnyxIcon :icon="minus" />
+        </span>
+        <OnyxLoadingIndicator v-if="props.loading" class="onyx-stepper__loading" type="circle" />
+        <input
+          v-else
+          v-model="value"
+          v-custom-validity
+          class="onyx-stepper__native"
+          type="number"
+          :aria-label="props.hideLabel ? props.label : undefined"
+          :autofocus="props.autofocus"
+          :disabled="props.disabled || props.loading"
+          :min="props.min"
+          :max="props.max"
+          :name="props.name"
+          :placeholder="props.placeholder"
+          :readonly="props.readonly"
+          :required="props.required"
+          :step="props.step"
+          :title="props.hideLabel ? props.label : undefined"
+          @change="handleChange"
+          @focus="emit('focus')"
+          @blur="emit('blur')"
+        />
+        <span
+          tabindex="0"
+          role="button"
+          class="onyx-stepper__counter"
+          :class="{ 'onyx-stepper__counter--disabled': props.max && props.max === value }"
+          :aria-label="t('add')"
+          @click="handleCounterButton($event, 'add')"
```

Comment:
We can use the native [`stepUp`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/stepUp) and [`stepDown`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/stepDown) function here:
```suggestion
          @click="inputRef?.stepUp()"
```